overlay/mkcrate.nix

```diff
@@ -188,10 +189,6 @@ let
     '';
 
     setBuildEnv = ''
-      isProcMacro="$( \
-        remarshal -if toml -of json Cargo.original.toml \
-        | jq -r 'if .lib."proc-macro" or .lib."proc_macro" then "1" else "" end' \
-      )"
```

Comment:
If we choose to add a fallback method for earlier versions of Rust, as described above, we will need to restore these removed lines for that fallback case.

Reply:
Done in c5f4107.